merico-dev/lake
# RefDiff plugin

# Summary

Calculates commits difference between 2 ref(branch/tag) and store into database

## step to verify

1. collect devlake data into database for testing by running `go run plugins/refdiff/utils/collect.og` in project root directory
2. trigger enrichment by creating a pipeline, check `README.md` for detail
3. verify `refs_commits_diffs` records against output of `git log <OLD_REF_NAME>..<NEW_REF_NAME>`

### Key Points



### Does this close any open issues?
Closes #1068
